internal/gather/gather_bytes_test.go

```diff
@@ -173,6 +182,85 @@ func TestGatherBytesReadSeeker(t *testing.T) {
 	require.Error(t, err)
 }
 
+func TestGatherBytesReaderAt(t *testing.T) {
+	// assert some preconditions that the reader conforms to ReaderAt
+	contentBuf := make([]byte, 1234567)
```

Comment:
As discussed with @Shrekster, we can probably make use of actual chunk size constants here. And we can iterate over a few key sizes, passing each into the iotest helper, just for an easy increase in coverage. I think @Shrekster will handle the quick enhancement, and I can approve.

Reply:
done